modules/volcano.R

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/Rscript
+args = commandArgs(trailingOnly = TRUE)
```

Comment:
What's the difference to the R script in the `bin` directory? Can this one be removed?